ui/src/components/pages/TaskDetailScreen.js

```diff
@@ -41,7 +41,14 @@ const TaskDetailScreen = (props) => {
             <div className="task-description">
               Description: {task.taskDescription}
             </div>
+            <div> <button
+              onClick={this.props.onDelete}
```

Comment:
we don't need the _this_ keyword in this circumstance, same with later, it knows props is referring to this component's props in particular. however, we might need the _this_ sent upwards if you are doing it this way:
```suggestion
              onClick={props.onDelete.bind(this, props.taskId)}
```